.github/workflows/release.yml

```diff
@@ -1,9 +1,16 @@
-name: Release
+name: Release and version
 
 on:
+  workflow_dispatch:
+    inputs:
+      branch:
+        description: "Trigger a release workflow"
+        type: string
+        default: "develop"
+        required: true
```

Comment:
Can be manually run too (though we woudln't need to manually run it). Manually running it can be necessary if networking flake happens for example